lib/modulesync.rb

```diff
@@ -120,31 +125,27 @@ def self.manage_module(puppet_module, module_files, module_options, defaults, op
 
     if options[:noop]
       Git.update_noop(module_name, options)
-    elsif !options[:offline]
+    elsif !options[:offline] && options[:pr]
```

Comment:
I believe the test failure is here. The `pushed` segment on 125/130 was previously run when `!offline`, now it's skipped when `!offline`, unless `pr` is enabled.